src/runtime/components/elements/Avatar.vue

```diff
@@ -32,6 +39,10 @@ export default defineComponent({
       type: String,
       default: null
     },
+    referrerpolicy: {
+      type: String as PropType<'no-referrer' | 'no-referrer-when-downgrade' | 'origin' | 'origin-when-cross-origin' | 'same-origin' | 'strict-origin' | 'strict-origin-when-cross-origin' | 'unsafe-url'>,
+      default: () => 'strict-origin-when-cross-origin'
```

Comment:
Why set a default here?

Reply:
Going off of the mdn docs. I just checked this out and looks like even when the props is null or undefined, the default is still part of the headers request.